usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -206,7 +206,7 @@ def parse_source(source, columns, download_job, working_dir, piid, assistance_id
         filters = request["filters"]
         if request.get("limit"):
             agency = ""
-        else:
```

Comment:
why is this change needed?

Reply:
https://federal-spending-transparency.atlassian.net/browse/DEV-1566?focusedCommentId=45266&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-45266